src/Orleans/Runtime/CallbackData.cs

```diff
@@ -69,21 +76,18 @@ public void StartTimer(TimeSpan time)
                 timeSinceIssued.Start();
             }
 
+            if (callbackTimeouter == null) callbackTimeouter = new CallbackTimeouter(config.ResendOnTimeout);
```

Comment:
Hmm, OK, but remember this is the thread static field, not just an instance field. I'm a little surprised that access is completely single threaded, but it could be, just wanted to give you a heads up so you can double  check